src/main.ts

```diff
@@ -31,20 +39,39 @@ async function run(): Promise<void> {
     const name = getInput('cluster_name', { required: true });
     const location = getInput('location', { required: true });
```

Comment:
Should we drop this as required since we now support grabbing it from resource name via `name` input?

Reply:
Done